examples/loading_spinners/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "loading_spinners"
+version = "0.1.0"
+authors = ["Nick Senger <dev@nsenger.com>"]
+edition = "2021"
+publish = false
+
+[dependencies]
+flo_curves = "0.7"
+iced = { path = "../..", features = ["canvas"] }
+iced_core = { path = "../../core" }
+iced_widget = { path = "../../widget" }
```

Comment:
You should be able to rely solely on `iced` if you enable the `advanced` feature, which will expose the `advanced` module.

Reply:
Oh cool, I also forgot to remove the flo_curves dependency now that it is instead using lyon for the easing curves. Updated in 56eacdb3583e701cde8cca074e151b24c3fd8df3